### ISSUE TITLE
Sound problem with game PCSG00895 

### ISSUE THREAD

**Issue by FrostMonarch95** [Vita3K/Vita3K#1243]
Some character sound are missing. And the log looks like this:
![image](https://user-images.githubusercontent.com/29565424/115814846-6f6d1900-a428-11eb-8e15-c4e8fa3ae7cc.png)
It seems that ffmpeg dose not decode at9 file so well. I extract the voice at9 file and use ffmpeg command line tool, same problem occurs 
![image](https://user-images.githubusercontent.com/29565424/115815091-e60a1680-a428-11eb-817f-e1a1eac10ca2.png)

**Comment by pent0** [comment]
It's convenient to use FFMPEG due to we use it in other places. But if it is also like this it may be not our problem to handle at all.

I want to confirm first if you extract this file by dumping from c++ code or extract game assets. I think FFMPEG is already well tested

**Comment by FrostMonarch95** [comment]
> It's convenient to use FFMPEG due to we use it in other places. But if it is also like this it may be not our problem to handle at all.
> 
> I want to confirm first if you extract this file by dumping from c++ code or extract game assets. I think FFMPEG is already well tested

I extract those at9 files from game assets. 
And I test decoding those at9 files with another tool https://www.psx-place.com/resources/at9-at3-converter.566/ and decoding seems fine.

**Comment by pent0** [comment]
You should report it to someone like @ TheAlexBarney, I think they handle the Atrac9 decoding implementation. This is not directly vita3k issue...

**Comment by Thealexbarney** [comment]
I actually didn't handle the ffmpeg implementation. I did the reverse engineering, documentation and reimplementation of the libatrac9 decoder. Might be worth trying my implementation on the audio file

**Comment by Zangetsu38** [comment]
iç no things is that, atrac9 is very special

**Comment by hobyst** [comment]
`.at9` decoding works just fine for me using FFmpeg 4.3.1 on Windows 10 using the following command:
```bash
ffmpeg -i input.at9 output.flac -vn -c:a flac -ar 44100 -sample_fmt s16 -ac 2
```

**Comment by pent0** [comment]
It depends on some files as being said.

**Comment by hobyst** [comment]
Oh, then they can use [MediaInfo](https://mediaarea.net/en/MediaInfo) to analyze the files. For comparison, use the `.at9` files found on `pd0:data/systembgm`, which are the ones I used to test it and decode just fine.

**Comment by FrostMonarch95** [comment]
https://drive.google.com/file/d/1gYnfmomdNkccG4gPh5hSHqPE41hYyIr8/view?usp=sharing

An example. Have a look. and very appreciate your help @pent0

**Comment by hobyst** [comment]
> https://drive.google.com/file/d/1gYnfmomdNkccG4gPh5hSHqPE41hYyIr8/view?usp=sharing
> 
> An example. Have a look. and very appreciate your help @pent0

Decodes just fine for me.

**Comment by FrostMonarch95** [comment]
> > https://drive.google.com/file/d/1gYnfmomdNkccG4gPh5hSHqPE41hYyIr8/view?usp=sharing
> > An example. Have a look. and very appreciate your help @pent0
> 
> Decodes just fine for me.

i test with your option and same problem occurs 
![image](https://user-images.githubusercontent.com/29565424/115840335-24fe9300-a44e-11eb-978e-24d10116b77c.png)

**Comment by hobyst** [comment]
Yeah, those errors pop up as well on my side, but the output file can be opened and plays just fine.

**Comment by FrostMonarch95** [comment]
> Yeah, those errors pop up as well on my side, but the output file can be opened and plays just fine.

Actually the character voice miss a word.

**Comment by Zangetsu38** [comment]
yes, akiba have this issue too, it start speek and get kdirectly last word, probvleme is no decode,is more timming etc rpcs3 have get this issue too old time

**Comment by pent0** [comment]
It's still see as FFMPEG issue

**Comment by Zangetsu38** [comment]
yes, but no relate directly it, ffmepg is fine, no probleme with it

**Comment by nishinji** [comment]
Outdated

**Comment by FrostMonarch95** [comment]
这是来自QQ邮箱的自动回复邮件。你好，我是王裕，已经收到你的邮件了。
